pkg/unittest/default_cluster.go

```diff
@@ -94,8 +94,8 @@ func DefaultAWSCluster() infrastructurev1alpha2.AWSCluster {
 	return cr
 }
 
-func DefaultCluster() capiv1alpha2.Cluster {
-	cluster := capiv1alpha2.Cluster{
+func DefaultCluster() *capiv1alpha2.Cluster {
```

Comment:
I had to change it otherwise it wouldn't satisfy the metv1.Object (GetAnnotation was the problem if not changing to a pointer). That's why testfiles which call this function needed to be changed.